src/browser/base/content/ZenStartup.mjs

```diff
@@ -39,6 +39,7 @@
         gZenUIManager.init();
         gZenVerticalTabsManager.init();
         gZenCompactModeManager.init();
+        gZenPinnedTabManager.init();
```

Comment:
We might benefit more from a preloaded script here. Run the script before the window is rendered. Checkout the keyboard shortcut's file to see how it's done